README.md

```diff
@@ -1 +1,93 @@
-# askup
\ No newline at end of file
+# Bridge for Adaptivity
```

Comment:
Fill it up please and point me to continue review

Reply:
done